README.md

```diff
@@ -93,72 +73,115 @@ npm install
 
 ## Usage
 
-### Generating a new key pair
+### Installing Support for Key Types
 
-Ed25519:
+In order to use this library, you will need to import and install driver
+libraries for key types you'll be working with via the `use()` method.
 
-```js
-const {Ed25519KeyPair} = require('crypto-ld');
+For example, to use this library with only the `ed25519` key type:
 
-const keyPair = await Ed25519KeyPair.generate();
 ```
+const {cryptoLd} = require('crypto-ld');
+cryptoLd.use(require('ed25519-key-pair');
 
-RSA:
+// With only one key type installed, you do not need to specify key type for
+// most operations
+const keyPair = await cryptoLd.generate(); // generates an ed25519 key pair
+```
 
-```js
-const {RSAKeyPair} = require('crypto-ld');
+To use the library with all supported key types:
 
-const keyPair = await RSAKeyPair.generate();
 ```
+const {cryptoLd} = require('crypto-ld');
```

Comment:
The `use()` API could be better updated to support "a-la-carte" installation of key types.

Proposed change: https://gist.github.com/gannan08/c1f8e9a0bf7a2df1062018a6799e8b3e

This was inspired by how VueJS libraries, like Vue Material, allow developers to either install the whole library or certain components. Link found [here](https://github.com/vuematerial/vue-material/blob/12dab986d3790e02dcae5693740308a79812369b/README.md#installation-and-usage). This is powered by VueJS' Plugin API that could be found [here](https://vuejs.org/v2/guide/plugins.html).

Reply:
I think the current design is _also_ aiming for "a-la-carte" approach. Just one level of indirection down -- on the key type level, not on the key suite with year level.

Reply:
Key type level does not suffice. There are use cases where applications may need to support multiple types of cryptographic key suites for a single key type. One example is when a ledger is updating from `Ed25519VerificationKey2018` based signatures to `Ed25519VerificationKey2020` based signatures.  How does the current design support this? 

Later down in my review is an example that may work.
https://github.com/digitalbazaar/crypto-ld/pull/47#discussion_r428956440

Reply:
@gannan08 I disagree. The whole point of our design is that only the latest suite will be used (although we do need to talk about migration strategies). But it's a major design goal that developers will not need to know the names of each individual suite - the point of abstraction is key type.